manual.py

```diff
@@ -0,0 +1,568 @@
+from cdp_backend.pipeline import ingestion_models
+from datetime import datetime
+
+event = ingestion_models.EventIngestionModel(
+    body=ingestion_models.Body("CITY OF HOUSTON . CITY COUNCIL", is_active=True),
+    sessions=[
+        ingestion_models.Session(
+            # session 0 no video, combine with session 1?????????????
+            video_uri="https://houstontx.new.swagit.com/videos/176697/embed",
+            session_index=1,
+            session_datetime='June 15, 2022'
+        )
+    ],
+    event_minutes_items = [
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("PRESENTATIONS")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("ROLL CALL AND ADOPT THE MINUTES OF THE PREVIOUS MEETING")
+        ),
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("MAYOR'S REPORT")
+        ),
+        # where should i put ACCEPT WORK???? following two belongs to accept work but not matters
+        ingestion_models.EventMinutesItem(
+            minutes_item = ingestion_models.MinutesItem("RECOMMENDATION from Director Houston Public Works for approval of final contract amount of $2,949,927.38 and acceptance of work on contract with REYTEC CONSTRUCTION RESOURCES, INC for FY2018 Drainage Rehab Work Orders #4 - 1.67% under the original Contract Amount - DISTRICTS A - PECK; B - JACKSON; C - KAMIN; D - EVANS-SHABAZZ; E - MARTIN; G - HUFFMAN; H - CISNEROS; I - GALLEGOS and K - CASTEX-TATUM")
```

Comment:
I figured out a method to get the name and the title of matters.

The name should be: "HPW 20CW10123 / Disclaimer of Harrisburg Alley/SY22-092" its the first text after the item creation date after clicking the link. These are usually pretty short. So I would use that as the name

The title is then the text you have here.